report_external_layout_simple/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Author: Silvija Butko. Copyright: JSC Focusate.
+# See LICENSE file for full copyright and licensing details.
+{
+    'name': "Report External Layout Simple",
+    'version': '14.0.3.0.0-rc.1',
+    'license': 'LGPL-3',
+    'summary': 'Report, External, Layout, Simple, Simplified',
+    'author': "Focusate",
+    'website': "http://www.focusate.eu",
+    'category': 'Hidden/Reporting',
+    'depends': [
+        # odoo
+        'web',
+    ],
+    'data': [
+        'views/assets.xml',
+        'views/report_templates.xml',
+        'data/report_paperformat_data.xml',
+        'data/report_layout_data.xml',
+    ],
+    'installable': True,
+}
```

Comment:
You should also check odoo.sh, because module was not installed and I had to do that manually.

Reply:
I guess thats some odoo.sh nonsense. Its set to install all modules in repo.